Add more details for installation via jbang

new to JBang and haven’t used it before, but the documentation seems incomplete. Simply doing the install and running jbang app install enola@enola-dev is not working, probably some catalog registration needs to be added first.

```
➜  ~ jbang app install enola@enola-dev

[jbang] [ERROR] Unknown catalog 'enola-dev'
[jbang] Run with --verbose for more details. The --verbose must be placed before the jbang command. I.e. jbang --verbose run [...]
```

cc: @vorburger

--- a/docs/agents.md
+++ b/docs/agents.md
@@ -18,7 +18,11 @@
 
 # Agents
 
+Try out any of our ready-to-use _**FREE**_ agents:
+
 * [Git Commit Message Agent](https://github.com/enola-dev/git-commit-message-agent)
+
+Or [Contact Us](support.md) to have us build an 🥷 Agent for YOU?
 
 Or _[make your own](tutorial/agents.md)!_
 

--- a/docs/specs/aiuri/index.md
+++ b/docs/specs/aiuri/index.md
@@ -56,7 +56,7 @@ This API has a very generous free tier. It requires a Google account.
 
 Note that the _Google (Gemini) AI API_ is a **different** API from the _Google Cloud Vertex AI Platform API._
 
-## Anthropic Claude
+### Anthropic Claude
 
 This API requires registering a credit card to purchase credits.
 

--- a/docs/support.md
+++ b/docs/support.md
@@ -20,4 +20,4 @@
 
 Depending on your needs, we may be able to offer commercial support for Enola.
 
-Please [contact us](mailto:michael.vorburger+enola.dev+support@gmail.com) to discuss.
+Please [contact us](mailto:support1@enola.dev) to discuss.

--- a/docs/use/index.md
+++ b/docs/use/index.md
@@ -68,13 +68,16 @@ PS: The [`enolac`](../download/latest/enolac) script contains the line above.
 
 ## JBang
 
-After [installing JBang](https://www.jbang.dev/download/), do:
+After [installing JBang](https://www.jbang.dev/download/), please [do](https://github.com/enola-dev/enola/issues/1641):
 
-    jbang app install enola@enola-dev
+    jbang catalog add --name enola-dev https://raw.githubusercontent.com/enola-dev/jbang-catalog/main/jbang-catalog.json
+    jbang --fresh app install enola@enola-dev
 
 You can now run Enola with:
 
     enola --help
+
+This runs it from `~/.jbang/bin`, which JBang [added](https://github.com/jbangdev/jbang/issues/2189) to your `PATH`.
 
 <!-- TODO Make this work... it doesn't quite, yet:
     enola server --chatPort=7070 --lm="google://?model=gemini-2.5-flash" --http-scheme --agents=https://raw.githubusercontent.com/enola-dev/enola/refs/heads/main/test/agents/chef-optimist.agent.yaml

